src/main/java/core/basesyntax/model/Machine.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.model;
+
+public abstract class Machine {
+    private String typeOfMmachine;
+
+    public String getTypeOfMmachine() {
+        return typeOfMmachine;
+    }
+
+    public void setTypeOfMmachine(String typeOfMmachine) {
+        this.typeOfMmachine = typeOfMmachine;
+    }
+
```

Comment:
remove this)  We don't need this here. It'll be enough if you print type of machines in theirs `System.out.println()`